Directory.Build.props

```diff
@@ -0,0 +1,39 @@
+<Project>
+
+  <PropertyGroup>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <LangVersion>latest</LangVersion>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Authors>Serilog Contributors</Authors>
+    <CurrentYear>$([System.DateTime]::Now.ToString(yyyy))</CurrentYear>
+    <Copyright>Copyright © Serilog Contributors 2017-$(CurrentYear)</Copyright>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageIcon>serilog-extension-nuget.png</PackageIcon>
+    <RepositoryType>git</RepositoryType>
+    <DebugType>embedded</DebugType>
```

Comment:
This will increase DLL size, which will negatively affect downstream consumers who ship apps with PDBs removed for size reasons.

Reply:
Or in terms of output dll: 18 KB vs 10KB for net core and 14KB vs 10KB for .NET targets. Does 4-8 KB make sense in 2024?

Reply:
Not all consuming scenarios are equivalent; Blazor and mobile apps often have tighter size requirements. If we did this in all Serilog projects, the additional 80% would feel like waste to me.

Reply:
So? Publish symbol package as well?

Reply:
I don't love SNUPKGs either but it seems like that's the option we're left with :-) :+1:

Reply:
done